pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -2,6 +2,7 @@
 from datetime import date, datetime, timedelta
 
 import pytest
```

Comment:
There is a fixture that need to add CalendarDay, which systematically tests lots of things

Reply:
This will exisit here since `CalendarDay` is in `offsets.__all__`

https://github.com/pandas-dev/pandas/blob/cf70d11a25ec59a5b630921d86759c7fa49884db/pandas/tests/tseries/offsets/conftest.py#L5-L6